internal/shared/version/version.go

```diff
@@ -29,7 +29,9 @@ func init() {
 	for _, setting := range info.Settings {
 		switch setting.Key {
 		case "vcs.revision":
-			gitCommit = setting.Value
+			if gitCommit == "" {
```

Comment:
Couldn't this be a check against `"unknown"`, and leave the default value as-is?

Reply:
i.e. is there a _need_ to change the default?

Reply:
@tmshort I am okay with leaving the default as it is, also `"unknown"` is _slightly better_ than empty string as it can be a bit ambiguous IMO.

Fixed in latest commit.